operator/pkg/gateway-api/cell.go

```diff
@@ -38,29 +41,27 @@ func (r gatewayApiConfig) Flags(flags *pflag.FlagSet) {
 	flags.String("gateway-api-secrets-namespace", r.GatewayAPISecretsNamespace, "Namespace having tls secrets used by CEC for Gateway API")
 }
 
-func registerController(lc hive.Lifecycle, config gatewayApiConfig) error {
+func registerController(lc hive.Lifecycle, config gatewayApiConfig, ctrlRuntimeManager ctrlRuntime.Manager, scheme *runtime.Scheme) error {
```

Comment:
(nit) probably also time for a `cell.In`.

Reply:
thanks, done